cs/unittest/TestJsonDirect.cs

```diff
@@ -149,7 +233,7 @@ public class JsonText
     [JsonObject(MemberSerialization = MemberSerialization.OptOut)]
     public class JsonContextArray
     {
-        [JsonProperty("_label")]
+        [JsonIgnore]
         public ILabel Label { get; set; }
```

Comment:
Actually “embedded” label support for JSON direct is not done yet.

You can pass labels directly with calls to Learn(), but extraction out of the object is not there yet…

Note: ingesting from string JSON, embedded labels are supported using _label.